packages/terra-demographics-banner/tests/nightwatch/demographics-banner-spec.js

```diff
@@ -0,0 +1,95 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
Oh I see. Looking back at the source code I see you are wrapping the demographics banner in a responsive element to handle the screen sizes. It might be a good idea to indicate in the documentation the demographics banner is dependent on the responsive element.